src/modules/dashboard/components/CoinMachine/CoinMachine.tsx

```diff
@@ -311,6 +346,7 @@ const CoinMachine = ({
             }}
             sellableToken={sellableToken}
             purchaseToken={purchaseToken}
+            periodTokens={periodTokens}
```

Comment:
I really that you put the period tokens back in here, as I worked really hard to refactor this so that it expressly doesn't use this specific value as it's not quite optimized.

However, I do understand why it's here, because you had to move the "Price next batch" display into the table.

I really don't like this, but this is a side-effect of just refactoring design willy nilly, you end up with content you need to move around, and you just slap it somewhere....

_(sorry for the rant :) )_

Reply:
I am not sure I understand why it's so bad as I just moved the `Price next batch` logic from one component to another and added one prop to `TokenSalesTable`...
I am happy to discuss this & refactor once I understand the ideal state you would like to see.